paymentsheet/src/main/java/com/stripe/android/paymentsheet/verticalmode/PaymentMethodRowButton.kt

```diff
@@ -31,47 +50,158 @@ internal fun PaymentMethodRowButton(
     subtitle: String?,
     onClick: () -> Unit,
     contentDescription: String? = null,
-    modifier: Modifier = Modifier,
+    modifier: Modifier = Modifier.fillMaxWidth(),
```

Comment:
Should this move below? It feels like we'd always want this max width, not optionally?